src/platform_impl/windows/window_state.rs

```diff
@@ -114,6 +114,12 @@ bitflags! {
         /// Drop shadow for undecorated windows.
         const MARKER_UNDECORATED_SHADOW = 1 << 20;
 
+        /// When minmizing a maximized Windows, `WM_SIZE` which we use to update the `MAXIMIZED` bit
+        /// is fired with `wparam` set to `SIZE_MINIMIZED` and thus the `MAXIMIZED` bit will be unset
+        /// and when un-minimizing the window, the `MAXIMIZED` bit will still be unset and later on
+        /// in `apply_diff` when `new.to_window_styles()` is called, it will not add `WS_MAXIMIZE` window style.
+        const MARKER_WAS_MAXIMIZED = 1 << 18;
```

Comment:
This conflicts with `IGNORE_CURSOR_EVENT`. Did you rebase without checking that the flag was still non-conflicting?

Reply:
yeah I forgot to check. Good catch! will fix it now.